package.json

```diff
@@ -14,7 +14,7 @@
   "scripts": {
     "lint": "tslint --project .",
     "build": "tsc --project .",
-    "test": "mocha ./test/index.test.js",
+    "test": "mocha ./test/index.test.js --exit",
```

Comment:
If this change was needed to make the tests exit, I think it indicates that the timers aren't being cleaned up properly -- which is a bug.

Reply:
Fixed by calling `disconnect` on the plugin in a couple of the tests to remove the timer.